app/src/main/java/com/sdp/movemeet/MainActivity.java

```diff
@@ -88,9 +76,9 @@ public void createDrawer(){
     public boolean onNavigationItemSelected(@NonNull MenuItem menuItem) {
         switch (menuItem.getItemId()) {
             case R.id.nav_home:
+                Navigation.goToHome(this.navigationView);
```

Comment:
Same as before, I think we don't need to make a redirection when we are already on the activity

Reply:
Same comment regarding adding the `finish()`